new/LIT7146Zar.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7146Zar" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ ሕማመ፡ ዛር፡ ወትግሪዳ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ṣalot baʾǝnta ḥǝmāma Zār wa-Tǝgridā</title>
+                <title xml:lang="en" corresp="#t1">Prayer against the harm of Zār and Tǝgridā</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BLorient12959"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><term key="Magic">Magic prayer</term> the harm of <term key="demon">Zār</term> and 
+                    <term key="demon">Tǝgridā</term> containing a sequence of Arabic or Islamic names or terms
+                    used for <term key="Asmat">ʾasmāt</term>, e.g. <foreign xml:lang="ar">ʾalāhumā</foreign> 
+                    (cf. <bibl><ptr target="bm:Griaule1930recettes"/><citedRange unit="page">111, 113</citedRange>
+                    <citedRange unit="item">CXVIII</citedRange></bibl>).
+                    The beginning of the prayer is <foreign xml:lang="gez">ጸሎት፡ በእንተ፡ ሕማመ፡ ዛር፡ ወትግሪዳ፡</foreign>, which is similar to the
+                    recurring phrase in prayers in 
+                    <ref type="mss" corresp="CamOr1885">Cambridge, Cambridge University Library Collections, CUL Or. 1885 , fol. 92v-93r</ref>
```

Comment:
Is it not better to refer to the manuscript without text in the element? And maybe "fols"?

Reply:
No. If the tag is empty it will print the title from the record: "Cambridge, Cambridge University Library Collections, CUL Or. 1885". If the tag is filled only with "fols. 92v-93r", than only "fols. 92v-93r" will be printed and the name or notification of the manuscript will not be visible. If both is desired or needed, it is necessary to write both in the tag, as far as I have understood.

Reply:
> No. If the tag is empty it will print the title from the record: "Cambridge, Cambridge University Library Collections, CUL Or. 1885". If the tag is filled only with "fols. 92v-93r", than only "fols. 92v-93r" will be printed and the name or notification of the manuscript will not be visible. If both is desired or needed, it is necessary to write both in the tag, as far as I have understood.

Correct. However if there is already an `msItem `in that manuscript with an `xml:id` you can refer to a specific `msItem ` (e.g. `CamOr1885#ms_i1`) and then do not need the text.